velox/substrait/VeloxToSubstraitType.h

```diff
@@ -28,12 +28,17 @@ class VeloxToSubstraitTypeConvertor {
   /// Convert Velox RowType to Substrait NamedStruct.
   const ::substrait::NamedStruct& toSubstraitNamedStruct(
       google::protobuf::Arena& arena,
-      const velox::RowTypePtr& rowType);
+      const velox::RowTypePtr& rowType,
+      std::vector<bool> nullableList);
 
   /// Convert Velox Type to Substrait Type.
   const ::substrait::Type& toSubstraitType(
       google::protobuf::Arena& arena,
-      const velox::TypePtr& type);
+      const velox::TypePtr& type,
+      bool nullable = true);
```

Comment:
It is necessary to have a default value for this parameter? If not, let's remove the default value for readability.